code/drasil-docLang/lib/Drasil/DocumentLanguage/Definitions.hs

```diff
@@ -83,7 +86,7 @@ makeDerivCons (E e) = EqnBlock e
 makeDerivCons s     = Paragraph s
 
 -- | Synonym for easy reading. Model rows are just 'String',['Contents'] pairs.
-type ModRow = [(String, [Contents])]
+type ModRow = [(Sentence, [Contents])]
```

Comment:
I don't know if we want the first bit of a `ModRow` to be a full `Sentence`. Why is this change needed?

Reply:
Also, these changes shouldn't be in this PR at all, they are about something other than duplicate 'the'.